packages/driver/cypress/support/utils.js

```diff
@@ -66,6 +66,12 @@ export const shouldBeCalledWithCount = (num) => (stub) => wrapped(stub).should('
 
 export const shouldNotBeCalled = (stub) => wrapped(stub).should('not.be.called')
 
+export const assertLogLength = (logs, expectedLength) => {
```

Comment:
Helper to log the list of logs received to help debug when log length is different than expected.